tests/fake-rule.ts

```diff
@@ -87,12 +99,6 @@ export default createTestingLibraryRule<Options, MessageIds>({
     return {
       'CallExpression Identifier': reportCallExpressionIdentifier,
       MemberExpression: reportMemberExpression,
-      'CallExpression > MemberExpression'(node: TSESTree.MemberExpression) {
```

Comment:
This was a silly report while implementing proper Testing Library utils detection. Now that are all implemented, it can be removed.